src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -458,18 +458,27 @@ private static function fixRequires(Tokens $tokens, int $index, Annotation $anno
         } elseif ('RequiresPhp' === $attributeName && isset($matches[3])) {
             $attributeTokens = [self::createEscapedStringToken($matches[2].' '.$matches[3])];
         } else {
-            $attributeTokens = [self::createEscapedStringToken($matches[2])];
+            $attributeTokens = [self::createEscapedStringToken(self::fixVersionConstraint($matches[2]))];
         }
 
         if (isset($matches[3]) && 'RequiresPhp' !== $attributeName) {
             $attributeTokens[] = new Token(',');
             $attributeTokens[] = new Token([T_WHITESPACE, ' ']);
-            $attributeTokens[] = self::createEscapedStringToken($matches[3]);
+            $attributeTokens[] = self::createEscapedStringToken(self::fixVersionConstraint($matches[3]));
         }
 
         return self::createAttributeTokens($tokens, $index, $attributeName, ...$attributeTokens);
     }
 
+    private static function fixVersionConstraint(string $version): string
+    {
+        if (Preg::match('/^[\d\.]+$/', $version)) {
```

Comment:
we shall cover non-stable releases too, like PHPUnit does, eg `5.4.0-alpha1`
ref https://github.com/sebastianbergmann/phpunit/blob/8.5.12/src/Util/Annotation/DocBlock.php#L33

Reply:
```suggestion
        if (Preg::match('/^\d/', $version)) {
```